CorsixTH/Lua/languages/english.lua

```diff
@@ -412,19 +447,19 @@ customise_window = {
   movies = "Global Movie Control",
   intro = "Play Intro Movie",
   paused = "Build on Paused",
-  volume = "Volume down hotkey",
+  volume = "Volume Down Hotkey",
   aliens = "Alien Patients",
   fractured_bones = "Fractured Bones",
-  average_contents = "Average contents",
+  average_contents = "Average Contents",
 }
 
 tooltip.customise_window = {
   movies = "Global movie control, this will allow you to disable all the movies",
   intro = "Turn off or on the intro movie, global movies will need to be on if you want the intro movie to play each time you load CorsixTH",
   paused = "In Theme Hospital the player would only be allowed to use the top menu if the game was paused. That is the default setting in CorsixTH too, but by turning this on everything is allowed while the game is paused",
-  volume = "If the volume down button is opening the casebook as well, turn this on to change the hotkey for the casebook to Shift + C",
-  aliens = "Because of the lack of proper animations we have by default made patients with Alien DNA so that they can only come from an emergency. To allow patients with Alien DNA to visit your hospital, other than by an emergency, turn this off",
-  fractured_bones = "Because of a poor animation we have by default made it so there are no patients with Fractured Bones that are female. To allow female patients with Fractured Bones to visit your hospital, turn this off",
+  volume = "If the volume down button is also opening the casebook, turn this on to change the hotkey for the casebook to Shift + C",
+  aliens = "Because of the lack of proper animations we have by default made patients with Alien DNA only come from an emergency. To allow patients with Alien DNA to visit your hospital, other than by an emergency, turn this off",
+  fractured_bones = "Because of some poor animations we have by default made it so there are no female patients with Fractured Bones. To allow female patients with Fractured Bones to visit your hospital, turn this off",
```

Comment:
Singular to plural changes the meaning. I'm not sure whether there was more than one bad / glitching animation.